### PR TITLE
Fix performance issue for history

### PR DESCRIPTION
**Pull request by akinny**
- added max limit to history
- made max history configurable
- default ui to 10,000 records
- Remove History from get Group & Webserver calls
- Sorry about the line endings in JpaGroupBuilder, removed getHistory

Before you contribute, please review these guidelines to help ensure a smooth process for everyone.

Thanks.

* Read [how to properly contribute to open source projects on Github][1].
* Fork the project.
* Use a feature branch.
* Write [good commit messages][2].
* Use the same coding conventions as the rest of the project.
* Commit locally and push to your fork until you are happy with your contribution.
* Make sure to add tests and verify all the tests are passing when merging upstream.
* Add an entry to the [Changelog][3] accordingly.
* Please add your name to the [CONTRIBUTORS.md][7] file. Adding your name to the [CONTRIBUTORS.md][7] file signifies agreement to all rights and reservations provided by the [License][4].
* [Squash related commits together][5].
* Open a [pull request][6].
* The pull request will be reviewed by the community and merged by the project committers.

[1]: http://gun.io/blog/how-to-github-fork-branch-and-pull-request
[2]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[3]: ./CHANGELOG.md
[4]: ./LICENSE
[5]: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
[6]: https://help.github.com/articles/using-pull-requests
[7]: ./CONTRIBUTORS.md

### CODE REVIEW

**Review by PeterTeunissen — APPROVED** [review]
ok

File: jwala-common/src/main/java/com/cerner/jwala/common/properties/PropertyKeys.java [hunk 1]
```diff
@@ -27,7 +27,8 @@
     JSCH_CHANNEL_SHELL_READ_INPUT_SLEEP_DURATION("jsch.channel.shell.read.input.sleep.duration"),
     JSCH_EXEC_READ_REMOTE_OUTPUT_LOOP_SLEEP_TIME("jsch.exec.read.remote.output.loop.sleep.time"),
     JSCH_SHELL_READ_REMOTE_OUTPUT_TIMEOUT("jsch.shell.read.remote.output.timeout"),
-    JSCH_EXEC_READ_REMOTE_OUTPUT_TIMEOUT("jsch.exec.read.remote.output.timeout");
+    JSCH_EXEC_READ_REMOTE_OUTPUT_TIMEOUT("jsch.exec.read.remote.output.timeout"),
+    MAX_JWALA_HISTORY("max.jwala.history");
```
Comment (jmahmoodcc) [inline jwala-common/src/main/java/com/cerner/jwala/common/properties/PropertyKeys.java]: Can we standardize that all of the properties specifically for jwala start with jwala?
So this property key would change to "jwala.max.history".

File: jwala-webapp/src/main/webapp/resources/js/react/group-operations/CommandStatusWidget.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ var CommandStatusWidget = React.createClass({
     },
     render: function() {
         var self = this;
-
+        this.props.numofRec=0;
```
Comment (jmahmoodcc) [inline jwala-webapp/src/main/webapp/resources/js/react/group-operations/CommandStatusWidget.js:10]: You can add the max.jwala.history property to ApplicationPropertySourceDefinition and that way it gets read from vars.properties and applied in the UI

**Review by jmahmoodcc — CHANGES_REQUESTED** [review]
looks good, I would just change the name of the property and associate the numRecs variable in the UI to the vars.properties using ApplicationPropertySourceDefinition

File: jwala-webapp/src/main/webapp/resources/js/react/group-operations/CommandStatusWidget.js (hunk 2 above)
Comment (akinny) [reply jwala-webapp/src/main/webapp/resources/js/react/group-operations/CommandStatusWidget.js:10]: done!

File: jwala-common/src/main/java/com/cerner/jwala/common/properties/PropertyKeys.java (hunk 1 above)
Comment (akinny) [reply jwala-common/src/main/java/com/cerner/jwala/common/properties/PropertyKeys.java]: I renamed it

**Review by jmahmoodcc — APPROVED** [review]

**Pull request merged**